metaserver/src/main/scala/scala/meta/languageserver/storage/Bytes.scala

```diff
@@ -0,0 +1,25 @@
+package scala.meta.languageserver.storage
+
+import java.nio.charset.StandardCharsets
+
+trait Bytes[A] { self =>
+  def fromBytes(bytes: Array[Byte]): A
+  def toBytes(e: A): Array[Byte]
+  def map[B](f: A => B, g: B => A): Bytes[B] = new Bytes[B] {
```

Comment:
This is not the signature I expect from map.

Reply:
I agree 😅 it changed to something more familiar in a later commit